src/Uno.UI/UI/Xaml/FrameworkElement.cs

```diff
@@ -119,6 +108,30 @@ Windows.UI.Xaml.ResourceDictionary Resources
 #endif
 		DependencyObject Parent => ((IDependencyObjectStoreProvider)this).Store.Parent as DependencyObject;
 
+		private bool _isParsing;
+		/// <summary>
+		/// True if the element is in the process of being parsed from Xaml.
+		/// </summary>
+		/// <remarks>This property shouldn't be set from user code. It's public to allow being set from generated code.</remarks>
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public bool IsParsing
+		{
+			get => _isParsing;
+			set
+			{
+				if (!value)
+				{
+					throw new InvalidOperationException($"{nameof(IsParsing)} should never be set from user code.");
+				}
+
+				_isParsing = value;
+				if (_isParsing)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this condition so that it does not always evaluate to 'true'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)